src/qt/locale/bitcoin_nl.ts

```diff
@@ -4421,75 +4421,75 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>Bitcoin Core and Dogecoin Core</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Bitcoin Core en Dogecoin Core</translation>
     </message>
     <message>
         <source>Amount under which a transaction output is considered dust, in %s (default: %s)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Bedrag waaronder een transactie-output als stof wordt beschouwd, in %s (default: %s)</translation>
     </message>
     <message>
         <source>Execute command when the best block changes (%s in cmd is replaced by block hash, %i is replaced by block number)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Commando uitvoeren wanneer het beste blok verandert (%s in cmd is replaced by block hash, %i is replaced by block number)</translation>
```

Comment:
`(%s in cmd is replaced by block hash, %i is replaced by block number)` => `(%s in cmd wordt vervangen door de blok hash, %I wordt vervangen door het blok nummer)`